### PR TITLE
Content.fix: minor updates to Staking docs

### PR DESCRIPTION
**Pull request by jimmy3dita**

### CODE REVIEW

**Comment by jimmy3dita** [comment]
@thisisjoshford we have the usual 404 bug with Dokia's platform, do you think we can merge it anyway?

**Comment by thisisjoshford** [comment]
> @thisisjoshford we have the usual 404 bug with Dokia's platform, do you think we can merge it anyway?
Yes, this is fine. We have decided to add this to the ignore list for now.

File: docs/validator/staking.md [hunk 1]
```diff
@@ -146,23 +228,28 @@ The 0/0/40 shows the total validators: connected peers / up to date peers / my p
...
-  amount: '100011163887239132720351',
-  code_hash: '11111111111111111111111111111111',
-  locked: '97985903901882082761',
-  storage_paid_at: 25,
-  storage_usage: 182
-}
-```
+<blockquote class="warning">
+    <strong>Heads Up!</strong><br><br>
+    You have to wait at least 2 epochs (43,200 blocks each) before your
```
Comment (he7d3r) [inline docs/validator/staking.md:243]: @stefanopepe aren't we missing a word after "your"? The sentence seems to end abruptly.

Comment (stefanopepe) [reply docs/validator/staking.md:243]: I wrote this one a while back, yes it's possible that on some edit I cut too much text. Feel free to submit a PR for this one!

Comment (he7d3r) [reply docs/validator/staking.md:243]: @stefanopepe here is an attempt to fix it: https://github.com/near/docs/pull/839. Let me know if it was about something else. Best regards

**Comment by stefanopepe** [comment]
@he7d3r I just realized that I've never come back to you. Thank you for this work, it's really appreciated!
Looping in @janewang as she's now the coordinator of the node experience and is taking care of validators engagement

**Comment by janewang** [comment]
Thanks @he7d3r  for this contribution. I see the PR is merged. Cheers!

**Pull request merged**